FrontEnd/Hopsy/src/screens/HomeScreen.js

```diff
@@ -91,6 +53,43 @@ export default class Test extends React.Component {
   }
 }
 
+posts = [{
```

Comment:
why was this deleted and moved down here?

Reply:
For visibility. Having it at the top of the file distracted from actual code